server/schemas/request.graphql

```diff
@@ -0,0 +1,106 @@
+type Request implements Node {
+  id: ID!
+  items: [RequestItem!]!
+  title: String!
+  description: String
+  createdBy: ID!
+  workspaceId: ID!
+  projectId: ID!
+  threadId: ID!
+  reviewersId: [ID!]!
+  state: RequestState!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+  approvedAt: DateTime
+  closedAt: DateTime
+  thread: Thread
+  workspace: Workspace
+  project: Project
+  reviewers: [User!]!
+}
+
+type RequestItem {
+  itemId: ID!
+  version: String!
```

Comment:
If a new item is updated after the request is created, does the version referenced by the request need to automatically switch to the latest version? If so, add a ref as well.

```suggestion
  version: String
  ref: String
```